code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -125,27 +118,15 @@
 					build_step++
 
 		if(ASSEMBLY_FIFTH_STEP)
-			switch(lasercolor)
-				if("b")
-					if(!istype(W, /obj/item/clothing/head/helmet/bluetaghelm))
-						return
-
-				if("r")
-					if(!istype(W, /obj/item/clothing/head/helmet/redtaghelm))
-						return
-
-				if("")
-					if(!istype(W, /obj/item/clothing/head/helmet))
-						return
-
-			if(!user.temporarilyRemoveItemFromInventory(W))
-				return
-			to_chat(user, "<span class='notice'>You add [W] to [src].</span>")
-			qdel(W)
-			name = "covered and shielded frame assembly"
-			item_state = "[lasercolor]ed209_hat"
-			icon_state = "[lasercolor]ed209_hat"
-			build_step++
+			if(istype(W, /obj/item/clothing/head/helmet))
+				if(!user.temporarilyRemoveItemFromInventory(W))
+					return
+				to_chat(user, "<span class='notice'>You add [W] to [src].</span>")
+				qdel(W)
+				name = "covered and shielded frame assembly"
+				item_state = "ed209_hat"
+				icon_state = "ed209_hat"
+				build_step++
 
 		if(5)
```

Comment:
Add defines for the rest of the assembly steps just for the sake of consistency. 
This makes me cry.

Reply:
😿 

adjusted in 0b185a1